java/src/apps/ConfigBundle.java

```diff
@@ -22,7 +22,7 @@
  * @version $Revision: 17977 $
  * @since 3.3.1
  */
-public class ConfigBundle extends apps.Bundle {
+public class ConfigBundle extends jmri.Bundle {
```

Comment:
This must extend apps.Bundle.
This bundle inherits alls strings from the Bundle that it extends, as well as its own, and directly extending jmri.Bundle prevents using classes from showing any strings in the apps.AppsBundle.properties file. Since this Bundle explicitly adds the strings in a second bundle file (not a normal situation) it should inherit from the Bundle in its own package.